wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingPerformanceTest.java

```diff
@@ -105,7 +115,46 @@ public void test_switchReaderWriterConnection() throws SQLException, IOException
     doWritePerfDataToFile(
         String.format(
             "./build/reports/tests/"
-            + "DbEngine_%s_Driver_%s_ReadWriteSplittingPerformanceResults_SwitchReaderWriterConnection.xlsx",
+                + "DbEngine_%s_Driver_%s_ReadWriteSplittingPerformanceResults_SwitchReaderWriterConnection.xlsx",
+            TestEnvironment.getCurrent().getInfo().getRequest().getDatabaseEngine(),
+            TestEnvironment.getCurrent().getCurrentDriver())
+    );
+
+    setReadOnlyPerfDataList.clear();
+
+    // internal connection pool results
+    final long connPoolSwitchToReaderMinOverhead =
+        resultsWithPools.switchToReaderMin - resultsWithoutPlugin.switchToReaderMin;
+    final long connPoolSwitchToReaderMaxOverhead =
+        resultsWithPools.switchToReaderMax - resultsWithoutPlugin.switchToReaderMax;
+    final long connPoolSwitchToReaderAvgOverhead =
+        resultsWithPools.switchToReaderAvg - resultsWithoutPlugin.switchToReaderAvg;
+
+    final long connPoolSwitchToWriterMinOverhead =
+        resultsWithPools.switchToWriterMin - resultsWithoutPlugin.switchToWriterMin;
+    final long connPoolSwitchToWriterMaxOverhead =
+        resultsWithPools.switchToWriterMax - resultsWithoutPlugin.switchToWriterMax;
+    final long connPoolSwitchToWriterAvgOverhead =
+        resultsWithPools.switchToWriterAvg - resultsWithoutPlugin.switchToWriterAvg;
+
+    final PerfStatSwitchConnection connPoolsConnectReaderData = new PerfStatSwitchConnection();
+    connPoolsConnectReaderData.connectionSwitch = "Switch to reader (open new connection)";
```

Comment:
With connection pools enabled, we sometimes connect to a new reader and sometimes reuse a reader connection from the internal pool. Let's maybe remove the "(open new connection)" from the label